code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm

```diff
@@ -14,7 +14,7 @@
 	melee_damage_upper = 10
 	poison_per_bite = 2
 	var/atom/cocoon_target
-	poison_type = "stoxin"
+	poison_type = "toxin"
```

Comment:
"pararein" is normal spider poison, did Reree ask for toxin instead of spider toxin?

Reply:
ahhh I wanted nurse to be better due to lower damage